tests/scenarios.go

```diff
@@ -305,11 +309,7 @@ func GetRequestsUndo(ctx gobdd.Context, operationID string) (func(interface{}) f
 				in[i] = object
 			}
 			request := undoOperation.Call(in)[0]
-			result := request.MethodByName("Execute").Call(nil)
-
-			if result[len(result)-1].Interface() != nil {
-				fmt.Printf("error in undo %v", result[len(result)-1])
-			}
+			request.MethodByName("Execute").Call(nil)
```

Comment:
we should try to add `t.Logf("error in undo %v", result[len(result)-1])`

Reply:
I don't have access to `t` here, should I just use the log package?

Reply:
using log will print to stderr, which will fail trigger a failure in gotestsum, not sure we want that